ios/Exponent.xcodeproj/xcshareddata/xcschemes/Exponent.xcscheme

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "0940"
-   version = "1.3">
+   version = "1.7">
```

Comment:
🤔

Reply:
It's been automatically changed by Xcode when I changed the default location for simulator, I guess it shouldn't break anything 😛